notebooks/misc/dp_mixgauss_truncated.ipynb

```diff
@@ -0,0 +1,740 @@
+{
```

Comment:
<p>There seems to be an issue with the latex</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/924/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>